Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/SliderSounds.cs

```diff
@@ -109,6 +109,14 @@ private void OnInteractionStarted(SliderEventData arg0)
                 grabReleaseAudioSource.PlayOneShot(interactionStartSound);
             }
         }
+
+        public void SetTickEvery(float value)
```

Comment:
This feels like it could be a [public property](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#do-9) for the private serialized field.

Also, please be sure to [add a summary](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#function--method-summary-headers)!